api-reference/beta/resources/application.md

```diff
@@ -18,82 +18,70 @@ Here is a JSON representation of the resource
-|groupMembershipClaims|String|A bitmask that configures the "groups" claim issued in a user or OAuth 2.0 access token that the application expects. The bitmask values are: 0: None, 1: Security groups and Azure AD roles, 2: Reserved, and 4: Reserved. Setting the bitmask to 7 will get all of the security groups, distribution groups, and Azure AD directory roles that the signed-in user is a member of. |
-|homepage|String|The URL to the application's homepage.|
-|identifierUris|String collection|The URIs that identify the application. For more information see, [Application Objects and Service Principal Objects](https://azure.microsoft.com/en-us/documentation/articles/active-directory-application-objects/). The **any** operator is required for filter expressions on multi-valued properties. Not nullable. |
-|keyCredentials|[keyCredential](keycredential.md) collection|The collection of key credentials associated with the application Not nullable.|
-|knownClientApplications|Guid collection|Client applications that are tied to this resource application. Consent to any of the known client applications will result in implicit consent to the resource application through a combined consent dialog (showing the OAuth permission scopes required by the client and the resource). Not nullable.|
-|logoutUrl|String|                              |
-|mainLogo|Stream|The main logo for the application. Not nullable|
-|oauth2AllowImplicitFlow|Boolean|Specifies whether this web application can request OAuth2.0 implicit flow tokens. The default is **false**. Not nullable.|
-|oauth2AllowUrlPathMatching|Boolean|Specifies whether, as part of OAuth 2.0 token requests, Azure AD will allow path matching of the redirect URI against the application's **replyUrls**. The default is **false**. Not nullable.|
-|oauth2Permissions|[oAuth2Permission](oauth2permission.md) collection|The collection of OAuth 2.0 permission scopes that the web API (resource) application exposes to client applications. These permission scopes may be granted to client applications during consent. Not nullable.|
-|oauth2RequirePostResponse|Boolean||
-|id|String|The unique identifier for the application. Inherited from [directoryObject](directoryobject.md). Key. Not nullable. Read-only.|
-|onPremisesPublishing|[onPremisesPublishing](onpremisespublishing.md)| The on-premises publishing settings used with Azure AD Application Proxy.| 
+|allowPublicClient|Boolean| |
+|api|[api](api.md)| Specifies settings for a Web API application. |
+|appRoles|[appRole](approle.md) collection|The collection of application roles that an application may declare. These roles can be assigned to users, groups, or service principals. Not nullable.|
+|applicationAliases|String collection| |
+|createdDateTime|DateTimeOffset| |
+|displayName|String|The display name for the application. |
+|id|String|The unique identifier for the application. Inherited from [directoryObject](directoryobject.md). Key. Not nullable. Read-only. |
```

Comment:
I don't think the inherited bit is correct.  Check with Suresh, but you might also want to point out that this is the same as the clientId/appId that gets generated during application registration.